src/controllers/login.js

```diff
@@ -0,0 +1,10 @@
+/* Logs the user in with email and password */
+const login = (req, res) => {
+  // eslint-disable-next-line
+  const { email, password } = req.body;
+
+  // TODO: Get user auth token from Firebase
```

Comment:
Here you'll wanna grab the auth token for the user by using their email and password, then send that token back to the frontend so the client can save that token in their cookies.

Reply:
For now I think we can remove this and use client side routing on the editor page